profiles/ksoe-codes.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://avatars.githubusercontent.com/u/108216630?v=4",
```

Comment:
Hmm, maybe you should add another picture of your profile? What do you think?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Oh sorry but i still don't want to add a profile picture, would be happy to add it soon :))
Regards ✨